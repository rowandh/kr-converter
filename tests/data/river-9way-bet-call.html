
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-86756675</p></td>
                                <td height="26" ><p align="center">&nbsp;2024-12-31 오전 6:57:16</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>hjagjhgdj (7 풀 하우스)</b></p></td>
<!--								<td ><p align="center">&nbsp;37880</p></td>   -->
								<td ><p align="center">&nbsp;37,880</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>fgyhnmr3</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠J <font color='red'>◆</font>10 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 [7 원 페어]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:256,674원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[fgyhnmr3]
<br>* 앤티: -1,000원(255,674원)
<br>* 홀 카드딜: ♠J(10) <font color='red'>◆</font>10(22) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♠J <font color='red'>◆</font>10)]
<br>* 베팅: 콜 -1,000원(254,674원) - 베팅순서: [0][3]
[2531ms]
<br>* 커뮤니티 카드 딜: H(♠J<font color='red'>◆</font>10) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠J <font color='red'>◆</font>10 ♣2)]
<br>* 베팅: 체크 -0원(254,674원) - 베팅순서: [1][4]
[2607ms]
<br>* 커뮤니티 카드 딜: H(♠J<font color='red'>◆</font>10) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠J <font color='red'>◆</font>10 <font color='red'>◆</font>3)]
<br>* 베팅: 다이 [0](254,674원) - 베팅순서: [2][4]
[2490ms]
<br>* 커뮤니티 카드 딜: H(♠J<font color='red'>◆</font>10) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[254,674원]
<br>* 결과: 패배 [족보:7 원 페어] [카드:<font color='red'>♥</font>7 ♣7 ♠J <font color='red'>◆</font>10 <font color='red'>◆</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">254,674</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>vcbbvasddas</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠10 ♣8 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 [7 원 페어]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:964,716원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[wowodjdn]
<br>* 앤티: -1,000원(963,716원)
<br>* 홀 카드딜: ♠10(9) ♣8(46) [10 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:10 탑(♠10 ♣8)]
<br>* 베팅: 콜 -1,000원(962,716원) - 베팅순서: [0][5]
[1242ms]
<br>* 커뮤니티 카드 딜: H(♠10♣8) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠10 ♣8 ♣2)]
<br>* 베팅: 체크 -0원(962,716원) - 베팅순서: [1][5]
[1774ms]
<br>* 커뮤니티 카드 딜: H(♠10♣8) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠10 ♣8 <font color='red'>◆</font>3)]
<br>* 베팅: 다이 [1](962,716원) - 베팅순서: [2][5]
[7218ms]
<br>* 커뮤니티 카드 딜: H(♠10♣8) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[962,716원]
<br>* 결과: 패배 [족보:7 원 페어] [카드:<font color='red'>♥</font>7 ♣7 ♠10 ♣8 <font color='red'>◆</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">962,716</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>kollosart</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>5 <font color='red'>◆</font>2 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 <font color='red'>◆</font>7 [7 풀 하우스]  (Small Blind)
<br>* 시작 : [StageNo:86756675] [Credit:153,967원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[kollosart]
<br>* 앤티: -1,000원(152,967원)
<br>* 홀 카드딜: <font color='red'>♥</font>5(30) <font color='red'>◆</font>2(14) [5 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:5 탑(<font color='red'>♥</font>5 <font color='red'>◆</font>2)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:151,967원]
<br>* 베팅: 체크 -0원(151,967원) - 베팅순서: [0][8]
[3270ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>5<font color='red'>◆</font>2) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 2 투 페어(<font color='red'>♥</font>7 ♣7 ♣2 <font color='red'>◆</font>2 <font color='red'>♥</font>5)]
<br>* 베팅: 체크 -0원(151,967원) - 베팅순서: [1][1]
[1448ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>5<font color='red'>◆</font>2) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 2 투 페어(<font color='red'>♥</font>7 ♣7 ♣2 <font color='red'>◆</font>2 <font color='red'>♥</font>5)]
<br>* 베팅: 체크 -0원(151,967원) - 베팅순서: [2][1]
[4032ms]
<br>* 베팅: 콜 -8,000원(143,967원) - 베팅순서: [2][8]
[3332ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>5<font color='red'>◆</font>2) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 턴 시작: [리버(3)] [족보:7 풀 하우스(<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 ♣2 <font color='red'>◆</font>2)]
<br>* 베팅: 체크 -0원(143,967원) - 베팅순서: [3][1]
[2077ms]
<br>* 종료: WinMoney[0원] Credit[143,967원]
<br>* 결과: 패배 [족보:7 풀 하우스] [카드:<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 ♣2 <font color='red'>◆</font>2]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">143,967</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>drsfbnuserfyb</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>J <font color='red'>♥</font>8 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 [7 원 페어]  (다이) (Big Blind)
<br>* 시작 : [StageNo:86756675] [Credit:366,080원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[qodfsznefij]
<br>* 앤티: -1,000원(365,080원)
<br>* 홀 카드딜: <font color='red'>♥</font>J(36) <font color='red'>♥</font>8(33) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>♥</font>J <font color='red'>♥</font>8)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:364,080원]
<br>* 베팅: 체크 -0원(364,080원) - 베팅순서: [0][9]
[341ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>8) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>♥</font>J <font color='red'>♥</font>8 ♣2)]
<br>* 베팅: 체크 -0원(364,080원) - 베팅순서: [1][2]
[1831ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>8) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>♥</font>J <font color='red'>♥</font>8 <font color='red'>◆</font>3)]
<br>* 베팅: 다이 [1](364,080원) - 베팅순서: [2][2]
[7371ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>8) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[364,080원]
<br>* 결과: 패배 [족보:7 원 페어] [카드:<font color='red'>♥</font>7 ♣7 <font color='red'>♥</font>J <font color='red'>♥</font>8 <font color='red'>◆</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">364,080</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>htytff</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣K ♠2 [K 탑]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:170,039원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:150]
<br>* NICKNAME:[htytff]
<br>* 앤티: -1,000원(169,039원)
<br>* 홀 카드딜: ♣K(51) ♠2(1) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♣K ♠2)]
<br>* 베팅: 다이 [0](169,039원) - 베팅순서: [0][4]
[3085ms]
<br>* 커뮤니티 카드 딜: H(♣K♠2) C (<font color='red'>♥</font>7♣2♣7)
<br>* 커뮤니티 카드 딜: H(♣K♠2) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 커뮤니티 카드 딜: H(♣K♠2) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[169,039원]
<br>* 결과: 패배 [족보:K 탑] [카드:♣K ♠2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">169,039</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>hjagjhgdj</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>6 ♣6 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 <font color='red'>◆</font>7 [7 풀 하우스]  
<br>* 시작 : [StageNo:86756675] [Credit:369,491원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[hjagjhgdj]
<br>* 앤티: -1,000원(368,491원)
<br>* 홀 카드딜: <font color='red'>◆</font>6(18) ♣6(44) [6 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:6 원 페어(<font color='red'>◆</font>6 ♣6)]
<br>* 베팅: 콜 -1,000원(367,491원) - 베팅순서: [0][6]
[318ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>6♣6) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 6 투 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>6 ♣6 ♣2)]
<br>* 베팅: 체크 -0원(367,491원) - 베팅순서: [1][6]
[202ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>6♣6) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 6 투 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>6 ♣6 <font color='red'>◆</font>3)]
<br>* 베팅: 콜 -8,000원(359,491원) - 베팅순서: [2][6]
[321ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>6♣6) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 턴 시작: [리버(3)] [족보:7 풀 하우스(<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 <font color='red'>◆</font>6 ♣6)]
<br>* 베팅: 체크 -0원(359,491원) - 베팅순서: [3][3]
[950ms]
<br>* 종료: WinMoney[37,880원] Credit[397,371원]
<br>* 결과: 승리 [족보:7 풀 하우스] [카드:<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 <font color='red'>◆</font>6 ♣6]

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>27,880</font></p></td>
											<td style="word-break:break-all"><p align="right">397,371</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>tblnj</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>9 ♠4 [9 탑]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:164,774원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:150]
<br>* NICKNAME:[tblnj]
<br>* 앤티: -1,000원(163,774원)
<br>* 홀 카드딜: <font color='red'>◆</font>9(21) ♠4(3) [9 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:9 탑(<font color='red'>◆</font>9 ♠4)]
<br>* 베팅: 다이 [0](163,774원) - 베팅순서: [0][1]
[1358ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (<font color='red'>♥</font>7♣2♣7)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>9♠4) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[163,774원]
<br>* 결과: 패배 [족보:9 탑] [카드:<font color='red'>◆</font>9 ♠4] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">163,774</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>rjrhrhh5</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>A <font color='red'>◆</font>Q <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 <font color='red'>◆</font>7 [7 트리플]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:85,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[rjrhrhh5]
<br>* 앤티: -1,000원(84,000원)
<br>* 홀 카드딜: <font color='red'>♥</font>A(26) <font color='red'>◆</font>Q(24) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>♥</font>A <font color='red'>◆</font>Q)]
<br>* 베팅: 콜 -1,000원(83,000원) - 베팅순서: [0][2]
[3672ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>A<font color='red'>◆</font>Q) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>♥</font>A <font color='red'>◆</font>Q ♣2)]
<br>* 베팅: 체크 -0원(83,000원) - 베팅순서: [1][3]
[1419ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>A<font color='red'>◆</font>Q) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 <font color='red'>♥</font>A <font color='red'>◆</font>Q <font color='red'>◆</font>3)]
<br>* 베팅: [하프] (8,000원) Credit(75,000원) - 베팅순서: [2][3]
[2376ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>A<font color='red'>◆</font>Q) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 턴 시작: [리버(3)] [족보:7 트리플(<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 <font color='red'>♥</font>A <font color='red'>◆</font>Q)]
<br>* 베팅: 다이 [1](75,000원) - 베팅순서: [3][2]
[7254ms]
<br>* 종료: WinMoney[0원] Credit[75,000원]
<br>* 결과: 패배 [족보:7 트리플] [카드:<font color='red'>♥</font>7 ♣7 <font color='red'>◆</font>7 <font color='red'>♥</font>A <font color='red'>◆</font>Q] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-10,000</font></p></td>
											<td style="word-break:break-all"><p align="right">75,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>315fasff</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠K <font color='red'>♥</font>4 <font color='red'>♥</font>7 ♣2 ♣7 <font color='red'>◆</font>3 [7 원 페어]  (다이) 
<br>* 시작 : [StageNo:86756675] [Credit:326,274원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[x9b4tq]
<br>* 앤티: -1,000원(325,274원)
<br>* 홀 카드딜: ♠K(12) <font color='red'>♥</font>4(29) [K 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:K 탑(♠K <font color='red'>♥</font>4)]
<br>* 베팅: 콜 -1,000원(324,274원) - 베팅순서: [0][7]
[4007ms]
<br>* 커뮤니티 카드 딜: H(♠K<font color='red'>♥</font>4) C (<font color='red'>♥</font>7♣2♣7)
<br>* 턴 시작: [플랍(1)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠K <font color='red'>♥</font>4 ♣2)]
<br>* 베팅: 체크 -0원(324,274원) - 베팅순서: [1][7]
[2512ms]
<br>* 커뮤니티 카드 딜: H(♠K<font color='red'>♥</font>4) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3)
<br>* 턴 시작: [턴(2)] [족보:7 원 페어(<font color='red'>♥</font>7 ♣7 ♠K <font color='red'>♥</font>4 <font color='red'>◆</font>3)]
<br>* 베팅: 다이 [0](324,274원) - 베팅순서: [2][7]
[651ms]
<br>* 커뮤니티 카드 딜: H(♠K<font color='red'>♥</font>4) C (<font color='red'>♥</font>7♣2♣7) (<font color='red'>◆</font>3) (<font color='red'>◆</font>7)
<br>* 종료: WinMoney[0원] Credit[324,274원]
<br>* 결과: 패배 [족보:7 원 페어] [카드:<font color='red'>♥</font>7 ♣7 ♠K <font color='red'>♥</font>4 <font color='red'>◆</font>3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-2,000</font></p></td>
											<td style="word-break:break-all"><p align="right">324,274</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-86756675">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


