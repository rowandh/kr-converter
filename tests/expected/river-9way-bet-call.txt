PokerStars Hand #15286756675:  Hold'em No Limit ($1000/$1000) - 2024/11/30 13:34:46 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: tblnj ($164774 in chips)
Seat 2: rjrhrhh5 ($85000 in chips)
Seat 3: fgyhnmr3 ($256674 in chips)
Seat 4: htytff ($170039 in chips)
Seat 5: vcbbvasddas ($964716 in chips)
Seat 6: hjagjhgdj ($369491 in chips)
Seat 7: 315fasff ($326274 in chips)
Seat 8: kollosart ($153967 in chips)
Seat 9: drsfbnuserfyb ($366080 in chips)
tblnj: posts the ante $1000
rjrhrhh5: posts the ante $1000
fgyhnmr3: posts the ante $1000
htytff: posts the ante $1000
vcbbvasddas: posts the ante $1000
hjagjhgdj: posts the ante $1000
315fasff: posts the ante $1000
kollosart: posts the ante $1000
drsfbnuserfyb: posts the ante $1000
kollosart: posts small blind $1000
drsfbnuserfyb: posts big blind $1000
*** HOLE CARDS ***
tblnj: folds
rjrhrhh5: calls $1000
fgyhnmr3: calls $1000
htytff: folds
vcbbvasddas: calls $1000
hjagjhgdj: calls $1000
315fasff: calls $1000
kollosart: checks
drsfbnuserfyb: checks
*** FLOP *** [7h 2c 7c]
kollosart: checks
drsfbnuserfyb: checks
rjrhrhh5: checks
fgyhnmr3: checks
vcbbvasddas: checks
hjagjhgdj: checks
315fasff: checks
*** TURN *** [7h 2c 7c] [3d]
kollosart: checks
drsfbnuserfyb: folds
rjrhrhh5: bets $8000
fgyhnmr3: folds
vcbbvasddas: folds
hjagjhgdj: calls $8000
315fasff: folds
kollosart: calls $8000
*** RIVER *** [7h 2c 7c] [3d] [7d]
kollosart: checks
rjrhrhh5: folds
hjagjhgdj: checks
hjagjhgdj collected $37880 from pot
*** SUMMARY ***
Total pot $40000 | Rake $2120
Board [7h 2c 7c 3d 7d]
Seat 1: tblnj mucked [9d 4s]
Seat 2: rjrhrhh5 mucked [Ah Qd]
Seat 3: fgyhnmr3 mucked [Js Td]
Seat 4: htytff mucked [Kc 2s]
Seat 5: vcbbvasddas mucked [Ts 8c]
Seat 6: hjagjhgdj showed [6d 6c] and won ($37880)
Seat 7: 315fasff mucked [Ks 4h]
Seat 8: kollosart showed [5h 2d] and lost
Seat 9: drsfbnuserfyb mucked [Jh 8h]